praw/models/reddit/comment.py

```diff
@@ -116,8 +147,7 @@ def submission(self, submission):
         """Update the Submission associated with the Comment."""
         submission._comments_by_id[self.name] = self
         self._submission = submission
-        # pylint: disable=not-an-iterable
-        for reply in getattr(self, "replies", []):
+        for reply in self.replies:
```

Comment:
Should this still be `self.replies` or `self._a.replies`?

Reply:
`self.replies` is accessing a model attribute here. The actual `replies` reddit attribute gets deleted [here](https://github.com/praw-dev/praw/blob/7e2fa34d63bf4ca1a37aa3c44b51482555938022/praw/models/reddit/comment.py#L180) since it’s not needed after its data has been objectified.

Side note, `self._data['replies']` should be preferred over `self._a.replies`.